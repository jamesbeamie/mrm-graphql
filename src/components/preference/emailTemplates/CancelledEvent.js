import React from 'react';
import '../../../assets/styles/cancelledEvent.scss';
import { convergeLogoIcon } from '../../../utils/images/images';

/**
 * Component that notifies a user about
 * an event that has been cancelled
 *
 * @returns {JSX}
 */
const cancelledEvent = () => (
  <div className="email">
    <div className="email-header">
      <div className="email-header-content">
        <img src={convergeLogoIcon} alt="" className="email-logo" />
        <p className="email-header-title">Converge</p>
      </div>
    </div>
    <div className="email-body">
      <p className="email-title">Cancelled Event</p>
      <div className="email-body-content">
        <p className="email-body-text">
          <span className="sync-title">Harrison &lt;&gt; Jolomi Sync</span>{' '}
          event was just cancelled
          <br /> and removed from your calendar by{' '}
          <span className="admin">Jolomi Otumara</span> with this note:
          <br />
          &quot;Cancelling because of our leadership offsite.&quot;
        </p>
      </div>
    </div>
    <div className="email-sub-footer">
      <div className="email-sub-footer-content">
        <span>Have a question?</span>
        <a href="/">support@converge.com</a>
      </div>
    </div>
    <p className="email-footer">
      You can change your <a href="/">subscription settings</a> anytime.
    </p>
  </div>
);

export default cancelledEvent;
