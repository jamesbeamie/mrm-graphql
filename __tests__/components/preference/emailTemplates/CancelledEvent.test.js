import React from 'react';
import { shallow } from 'enzyme';

import CancelledEvent from '../../../../src/components/preference/emailTemplates/CancelledEvent';

describe('Cancelled Event Component', () => {
  const wrapper = shallow(<CancelledEvent />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
