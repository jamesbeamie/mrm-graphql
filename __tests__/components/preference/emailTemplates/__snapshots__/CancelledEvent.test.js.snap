// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cancelled Event Component should render properly 1`] = `
<div
  className="email"
>
  <div
    className="email-header"
  >
    <div
      className="email-header-content"
    >
      <img
        alt=""
        className="email-logo"
        src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2Fconverge_logo.svg?alt=media&token=cd6a1244-91bb-4a1b-8d74-696bd42053ac"
      />
      <p
        className="email-header-title"
      >
        Converge
      </p>
    </div>
  </div>
  <div
    className="email-body"
  >
    <p
      className="email-title"
    >
      Cancelled Event
    </p>
    <div
      className="email-body-content"
    >
      <p
        className="email-body-text"
      >
        <span
          className="sync-title"
        >
          Harrison &lt;&gt; Jolomi Sync
        </span>
         
        event was just cancelled
        <br />
         and removed from your calendar by
         
        <span
          className="admin"
        >
          Jolomi Otumara
        </span>
         with this note:
        <br />
        "Cancelling because of our leadership offsite."
      </p>
    </div>
  </div>
  <div
    className="email-sub-footer"
  >
    <div
      className="email-sub-footer-content"
    >
      <span>
        Have a question?
      </span>
      <a
        href="/"
      >
        support@converge.com
      </a>
    </div>
  </div>
  <p
    className="email-footer"
  >
    You can change your 
    <a
      href="/"
    >
      subscription settings
    </a>
     anytime.
  </p>
</div>
`;
